app/controllers/frontend_log_controller.rb

```diff
@@ -27,6 +26,7 @@ class FrontendLogController < ApplicationController
     'IdV: download personal key' => :idv_personal_key_downloaded,
     'IdV: Native camera forced after failed attempts' => :idv_native_camera_forced,
     'Multi-Factor Authentication: download backup code' => :multi_factor_auth_backup_code_download,
+    'Show Password Button Clicked' => :show_password_button_clicked,
```

Comment:
This mapping is not working as expected, and the events are still being logged with the "Frontend:" prefix. I think it has to do with the difference in case, "Show Password Button Clicked" vs. "Show Password button clicked".

```suggestion
    'Show Password button clicked' => :show_password_button_clicked,
```